src/apple/project.rs

```diff
@@ -96,11 +96,16 @@ pub fn gen(
             map.insert("file-groups", &source_dirs);
             map.insert("ios-frameworks", metadata.ios().frameworks());
             map.insert("ios-vendor-frameworks", metadata.ios().vendor_frameworks());
+            map.insert("ios-vendor-sdks", metadata.ios().vendor_sdks());
             map.insert("macos-frameworks", metadata.macos().frameworks());
             map.insert(
                 "macos-vendor-frameworks",
```

Comment:
formatting inconsistency

(is your rustfmt working?)

Reply:
Apparently it wasn't. It's back now after restating tho.